packaging/nnstreamer.spec

```diff
@@ -1243,6 +1268,14 @@ cp -r result %{buildroot}%{_datadir}/nnstreamer/unittest/
 %{_prefix}/lib/nnstreamer/filters/libnnstreamer_filter_tvm.so
 %endif
 
+%if 0%{?tizen_hal_support}
+%files tizen-hal
+%manifest nnstreamer.manifest
+%defattr(-,root,root,-)
+%{_prefix}/lib/nnstreamer/filters/libnnstreamer_filter_snpe.so
+%{_prefix}/lib/nnstreamer/filters/libnnstreamer_filter_vivante.so
+%endif
```

Comment:
Can you really assume that every subplugin for Tizen will be available in Tizen-HAL build?